04/app.js

```diff
@@ -6,4 +6,25 @@ const user = {
         month: '04',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+const now = new Date();
+
+const calendar =function()
+{
+   console.log( now.getFullYear(),
+                now.getMonth()+1,
+                now.getDate());
+}
+
+calendar();
+
+user.check =function (b)
+{
+  console.log(this[b])
+  for(key in this[b])
+  {
+      console.log(this[b][key])
+  }
+}
```

Comment:
Tutaj coś nie działa. Wew. tej funkcji pobrałbym datę tj `const now = new Date();` oraz sprawdził czy `now.getDate() === this.born.day` i analogicznie dla miesiąca oraz roku. Jeśli tak to wtedy wiemy że są urodziny :)